apis/coralogix/v1alpha1/alert_types.go

```diff
@@ -1053,6 +1061,23 @@ func expandNotification(notification Notification) (*alerts.AlertNotification, e
 	return result, nil
 }
 
+func searchIntegrationID(ctx context.Context, name string) (uint32, error) {
```

Comment:
Just wondering if maybe makes sense for this function to return a integration struct, so in the future, we can leverage this method to manage integrations using the Operator.

Reply:
When we mange integrations we (probably) won't have to query all the webhooks